RedHatInsights/compliance-frontend
# feat(devtools): RHICOMPL-2292 - Prepare for running with federated modules

This will allow to run compliance and enable routes to import the federated modules exposed by inventory

(This assumes that https://github.com/RedHatInsights/insights-inventory-frontend/pull/1418 is merged)

## How to test

In order to verify and test this PR and verify that it is possible to run with federated modules locally

1. Clone the `insights-inventory-frontend` repo (if not already)
2. Apply changes to the `InventoryTable`[0] component
3. Follow the instructions added to the README in this PR
4. Verify that the changes applied to the `InventoryTable` component are visible within compliance.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist

[0] https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/src/components/InventoryTable/InventoryTable.js
